source/braille.py

```diff
@@ -2547,7 +2547,7 @@ def getSerialPorts(filterFunc=None):
 	@type filterFunc: callable
 	"""
 	if filterFunc and not callable(filterFunc):
-		raise ValueError("The provided filterFunc is not callable")
+		raise TypeError("The provided filterFunc is not callable")

```

Comment:
What is the reasoning for this change?

Reply:
addonHandler.getAvailableAddons now accepts a filter function as parameter, similar to here. However, when writing the new code for addonHandler, I discovered that this raised exception deals with a wrong type, not with a wrong value. Therefore it ought to be TypeError there, and I changed it here for consistency reasons.